draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx

```diff
@@ -43,11 +47,24 @@ export type TooltipProps = {
   isInitiallyVisible?: boolean
 }
 
+const positionToPlacement = new Map<Position, Placement>([
+  ["above", "top"],
+  ["below", "bottom"],
+  ["left", "left"],
+  ["right", "right"],
+])
+
 // Sync with Tooltip.scss
-const arrowHeight = 10
-const arrowWidth = 20
+const arrowHeight = 7
+const arrowWidth = 14
 
-const TooltipContent = ({ position, text, referenceElement, tooltipId }) => {
+const TooltipContent = ({
```

Comment:
I won't ask you to refactor this right now since I'm aiming to set up a discussion first, however, it is generally much cleaner if every component (subcomponents included) gets its own file. This will allow the file to be much smaller and therefore easier to read, including the scss. It can also encourage better testing depending on the component :)